src/Services/Masa.Auth.Service.Admin/Program.cs

```diff
@@ -27,6 +29,8 @@
 
 builder.Services.AddMasaRedisCache(builder.Configuration.GetSection("RedisConfig"));
 
+builder.Services.AddScoped<ISaveChangesFilter, SoftDeleteSaveChangesFilter>();
```

Comment:
Why did you add this manually?

Reply:
to fixed softdelete,when update contrib preview3 will delete this